<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela de permissões</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(62, 132, 238);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="InputUsuario">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Usuário:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSetar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(78, 154, 6);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Setar permissões</string>
    </property>
   </widget>
   <widget class="QLabel" name="IconeUsuario">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../GIT/HELLLO_WORLD/icone_usuario.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCancelar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(245, 121, 0);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string>Cadastro de permissões a usuário</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CbCriar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CbEditar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CbExcluir">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Permissões:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CbRoot">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Root</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>InputUsuario</tabstop>
  <tabstop>BtnSetar</tabstop>
  <tabstop>BtnCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
